packages/devtools_app/lib/src/shared/analytics/_analytics_web.dart

```diff
@@ -136,6 +138,8 @@ extension type GtagEventDevTools._(JSObject _) implements GtagEvent {
   external int? get root_set_count;
   external int? get row_count;
   external int? get inspector_tree_controller_id;
+  external String? get android_app_id;
+  external String? get ios_bundle_id;
```

Comment:
These will need to be added to the custom dimensions and metrics in the GA console for the legacy DevTools analytics if we want to track there. Otherwise they should automatically be included in the unified_analytics data for DevTools.